src/ReceiveAndProcessDataApp/receiveDataFile.cs

```diff
@@ -0,0 +1,70 @@
+using System.IO;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Newtonsoft.Json;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Collections.Generic;
+
+public static class BlobJsonTrigger
```

Comment:
Can we rename this class to ReceiveData and also rename the file to ReceiveData.cs

Reply:
sure working on those comments